operator/controllers/kyma_controller_remote_sync_test.go

```diff
@@ -45,19 +47,20 @@ var _ = Describe("Kyma with multiple module CRs in remote sync mode", Ordered, f
 
 	It("CR add from client should be synced in both clusters", func() {
 		By("Remote Kyma created")
-		Eventually(RemoteKymaExists(runtimeClient, kyma.GetName()), 30*time.Second, interval).Should(Succeed())
+		Eventually(RemoteKymaExists(runtimeClient, kyma.GetName()), 30*time.Second,
+			testutils.Interval).Should(Succeed())
 		remoteKyma, err := GetKyma(runtimeClient, kyma.GetName())
 		Expect(err).ShouldNot(HaveOccurred())
 
 		By("add skr-module-client to remoteKyma.spec.modules")
 		remoteKyma.Spec.Modules = []v1alpha1.Module{
 			*skrModuleFromClient,
 		}
-		Eventually(runtimeClient.Update(ctx, remoteKyma), timeout, interval).Should(Succeed())
+		Eventually(runtimeClient.Update(ctx, remoteKyma), testutils.Timeout, testutils.Interval).Should(Succeed())
```

Comment:
maybe we should import with `. "github.com/kyma-project/lifecycle-manager/operator/internal/testutils"` so that we dont have to use the package name everytime? What do you think? Then you dont need to adjust all tests here.

Reply:
The tests will anyways be adjusted by switching to an exported variable from test utils.